network/struct.go

```diff
@@ -185,6 +185,8 @@ type Packet struct {
 	Constructors protobuf.Constructors
 	// possible error during unmarshalling so that upper layer can know it
 	err error
+	// XXX REMOVE
+	To *ServerIdentity
```

Comment:
Looks like although it is written in two places, this field is never read. so, we could really remove it?